package.json

```diff
@@ -29,11 +29,11 @@
   },
   "peerDependencies": {
     "@emotion/core": "^10.0.28",
```

Comment:
Can we delete this now?

Reply:
good catch, yes this will be turned into `@emotion/react` shortly